PR-App/PR-App/Custom Views/Labels/CustomLabel.swift

```diff
@@ -19,16 +19,15 @@ class CustomCellLabel: UILabel {
         fatalError("init(coder:) has not been implemented")
     }
     
-    init(text: String) {
-        super.init(frame: .zero)
-        configure()
+    convenience init(text: String, alignment: NSTextAlignment, size: CGFloat, weight: UIFont.Weight, color: UIColor) {
```

Comment:
Ah, so that is the reason why You are using this long-ass constructor. I would not do it like that, because that it not a custom Label, it just passes arguments from initializer, so what is the point of it? That makes the usage it too complicated for me. But remember, that is only a comment not a some kind of ass-whooping mockery. Just my opinion.